rtloader/three/three.cpp

```diff
@@ -520,6 +520,7 @@ char **Three::getCheckWarnings(RtLoaderPyObject *check)
         if (warn == NULL) {
             setError("there was an error browsing 'warnings' list: " + _fetchPythonError());
 
+            // Is this a bug? ---> if warnings[jdx] != NULL and jdx > idx (uninitialized memory)
```

Comment:
Seems like it indeed ! We might want to fix this in another PR. I'll create a card in our backlog.